src/App.java

```diff
@@ -13,22 +13,111 @@ public void start() {
         printGreeting(userName);
         printInstructions();
         doMadlib();
-        System.out.println("hi");
+
     }
     
     /* Here you should create all missing methods used by the start() method
      * I have created the signature for ONE of the methods below.
      * You can create any other methods you need as well!
      */ 
-    
+   
+    public String getUserName() {
```

Comment:
This method does not have a comment explaiing what it does, as do none of the other methods youc reated